can_tools/scrapers/official/federal/CDC/cdc_county_cases_deaths.py

```diff
@@ -0,0 +1,61 @@
+from typing import List
+import pandas as pd
+from can_tools.scrapers.official.base import ETagCacheMixin, FederalDashboard
+from can_tools.scrapers import variables
+from multiprocessing import get_context
+import requests
+
+from can_tools.scrapers.base import ALL_STATES_PLUS_DC
+
+
+class CDCCountyCasesDeaths(FederalDashboard, ETagCacheMixin):
+    has_location = True
+    location_type = "county"
+    source = "https://data.cdc.gov/Case-Surveillance/Weekly-United-States-COVID-19-Cases-and-Deaths-by-/pwn4-m3yp"
+    source_name = "Centers for Disease Control and Prevention"
+    provider = "cdc"
+    fetch_url = "https://covid.cdc.gov/covid-data-tracker/COVIDData/getAjaxData?id=integrated_county_timeseries_fips_{county_fips}_external"
+
+    variables = {
+        "cumulative_cases": variables.CUMULATIVE_CASES_PEOPLE,
+        "cumulative_deaths": variables.CUMULATIVE_DEATHS_PEOPLE,
+    }
+
+    def __init__(self, execution_dt: pd.Timestamp = pd.Timestamp.utcnow()):
+        ETagCacheMixin.initialize_cache(
+            self, cache_url=self.fetch_url, cache_file="cdc_county_cases_deaths.txt"
+        )
+        super().__init__(execution_dt=execution_dt)
+
+    def fetch(self):
+        county_fips: List[str] = [
+            self._retrieve_counties(state=state, fips=True)
+            for state in ALL_STATES_PLUS_DC
+        ]
+        county_fips = [item.zfill(5) for sublist in county_fips for item in sublist]
+        with get_context("spawn").Pool(processes=16) as pool:
+            data = pool.map(self._fetch_county, county_fips)
+        return pd.concat(data, axis=0)
+
+    def normalize(self, data: pd.DataFrame) -> pd.DataFrame:
+        # Calculate cumulative cases and deaths from new counts.
```

Comment:
Calculating cumulative cases here, even though we'll eventually calculate "new cases" again later in the pipeline. I'm doing this to match the format that the NYT data in order to minimize the amount of downstream changes/updates